projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -270,19 +266,11 @@ deploymentVdkDistributionImage:
   registryUsernameReadOnly: ""
   registryPasswordReadOnly: ""
 
-  repository: quickstart-vdk
-  tag: "release"
-
 
-# The base image which will be used to create the image where data job would be run.
-# On top of it the job source and its dependencies are installed for each job.
-# It can be customized further by installing other native dependencies necessary to run different kinds of libraries
-# For example the below default image adds support for Oracle client on top of standard python image.
-# The SDK (vdk) is installed separate image (to enable zero time upgrades) see deploymentVdkDistributionImage
+## The registry URL and associated credentials needed to pull the data job base image.
+## These are stored in a kubernetes secret.
 deploymentDataJobBaseImage:
   registry: registry.hub.docker.com/versatiledatakit
```

Comment:
this can lead to pretty cryptic errors where the auth details is for a different registry than the registry specified below.

Reply:
This is a good point. I've created an issue for this, https://github.com/vmware/versatile-data-kit/issues/2100, where we can discuss possible approaches how to make the configuration more robust.